helm-crd.jsonnet

```diff
@@ -0,0 +1,44 @@
+// CRD and tiller, with controller running as a sidecar
+
+local kube = import "kube.libsonnet";
+local tiller = import "tiller.jsonnet";
```

Comment:
should this be `tiller-deployment`?

Reply:
actually it doesn't look like this var is being used at all here?

Reply:
oops, yep - copy+paste cruft from upstream helm-crd config.  Removed.